libbeat/management/status_string.go

```diff
@@ -0,0 +1,29 @@
+// Code generated by "stringer -type=Status"; DO NOT EDIT.
```

Comment:
The linter asks for a license header.

Reply:
:facepalm: generated code is such a pain.